Mage.Server.Plugins/Mage.Game.PillarOfTheParunsDuel/src/mage/game/PillarOfTheParunsDuel.java

```diff
@@ -0,0 +1,120 @@
+package mage.game;
+
+import mage.abilities.Ability;
+import mage.abilities.DelayedTriggeredAbility;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.ConjureCardEffect;
+import mage.constants.*;
+import mage.game.events.GameEvent;
+import mage.game.match.MatchType;
+import mage.game.mulligan.Mulligan;
+import mage.game.turn.TurnMod;
+import mage.players.Player;
+
+import java.util.UUID;
+
+/**
+ * @author Susucr
+ */
+public class PillarOfTheParunsDuel extends GameImpl {
```

Comment:
Must add game mode description, rules and other information here as docs/comments. If it's an official/public then must contains link to the public page/wiki.